core/test/functional/api/v2/admin/posts_spec.js

```diff
@@ -384,13 +383,19 @@ describe('Posts API V2', function () {
                         localUtils.API.checkResponse(res.body.posts[0], 'post');
                         should.not.exist(res.headers['x-cache-invalidate']);
 
-                        res.body.posts[0].title.should.eql(post.title);
-                        res.body.posts[0].status.should.eql(post.status);
-                        res.body.posts[0].published_at.should.eql('2016-05-30T07:00:00.000Z');
-                        res.body.posts[0].created_at.should.not.eql(post.created_at.toISOString());
-                        res.body.posts[0].updated_at.should.not.eql(post.updated_at.toISOString());
-                        res.body.posts[0].updated_by.should.not.eql(post.updated_by);
-                        res.body.posts[0].created_by.should.not.eql(post.created_by);
+                        return models.Post.findOne({
```

Comment:
This test is meant to check if includes are present, but no assertions around 'authors' or 'tags' being present are done here :man_shrugging:

Reply:
will check

Reply:
the assertions are still in place :)